plugins/standard_tests/pyproject.toml

```diff
@@ -8,8 +8,7 @@ packages = [{include = "newhelm"}]
 
 [tool.poetry.dependencies]
 python = "^3.10"
-# TODO: This probably shouldn't be here.
-perspective_api = {version = "0.1.1", optional = true}
+perspective_api = {version = "*"}
```

Comment:
Consider also running `poetry lock` to update the lock file.

Reply:
Oh, duh, right.

Reply:
Oops! Great point.